src/fastgltf_types.hpp

```diff
@@ -260,7 +260,7 @@ namespace fastgltf {
     }
 
     constexpr std::uint32_t getElementByteSize(AccessorType type, ComponentType componentType) noexcept {
-        return getNumComponents(type) * (getComponentBitSize(componentType) / 8);
+        return (getNumComponents(type) * getComponentBitSize(componentType) + 7u) / 8u;
```

Comment:
What's with this change?

Reply:
Probably not strictly necessary since all current glTF types are powers of 2, but this is the correct way to pack any sized components into a certain number of bytes. I felt the need to correct this since it would otherwise be weird to bother working in bit sizes for the components.